src/components/game/components/board/components/hand/Hand.js

```diff
@@ -1,59 +1,13 @@
 import { React } from "react";
 
-import { cardNumber, cardSuitChar } from "lib/handCheck";
 import Card from "../card";
+import {suitCharToSymbol,suitCharToString,cardNumtoRank,cardSuitChar,cardNumber} from "lib/cardFunctions";
 
 const Hand = ({ hand, visible, onClick, isSelected }) => {
-  const suitCharToSymbol = (suitChar) => {
-    switch (suitChar) {
-      case "H":
-        return <>&hearts;</>;
-      case "C":
-        return <>&clubs;</>;
-      case "D":
-        return <>&diams;</>;
-      case "S":
-        return <>&spades;</>;
-      default:
-        return null;
-    }
-  };
-
-  const suitCharToString = (suitChar) => {
-    switch (suitChar) {
-      case "H":
-        return "hearts";
-      case "C":
-        return "clubs";
-      case "D":
-        return "diams";
-      case "S":
-        return "spades";
-      default:
-        return null;
-    }
-  };
-
-  const cardNumtoRank = (suitRank) => {
-    if (suitRank <= 9) {
-      return suitRank.charAt(1).toString();
-    }
-    switch (suitRank) {
-      case "10":
-        return "10";
-      case "11":
-        return "J";
-      case "12":
-        return "Q";
-      case "13":
-        return "K";
-      case "14":
-        return "A";
-      default:
-        return null;
-    }
-  };
-
+ 
+  const cardSelector = (value) => {
+  return !value
+  }
```

Comment:
This is a bit cryptic. What is it supposed to do?

Reply:
Unused leftover, removed for now.